README.md

```diff
@@ -112,6 +112,15 @@ In `src/legendCreation.js`, add `addLayerNameURLPair(layer_var, "img_url");`, wh
 
 #### Add the layers browser menu from the demo page
 To add the layers browser menu from the demo page,
+
+Dependencies:
+```
+  <!-- Bootstrap --> 
+  <link rel="stylesheet" href="../node_modules\bootstrap\dist\css\bootstrap.min.css">
```

Comment:
@sagarpreet-chadha, this just occurred to me. To add these dependencies shouldn't I point it to LEL's node modules? So here, shouldn't I be writing "leaflet-environmental-layers/node_modules" instead to make sense? It wouldn't work otherwise unless the destination repo had bootstrap installed, right?

Reply:
Found that only LBL was in the node_modules folder in LEL when installing as a package. I was of the impression that all dependencies of a package would be included in the node_modules of that package. So I am assuming we should explicitly mention in the documentation that bootstrap needs to be installed? The new menu needs bootstrap load correctly.